test/fixtures/app.js

```diff
@@ -13,6 +13,32 @@ app.dataSource('mail', { connector: 'mail', defaultForType: 'mail' });
 // Attach all built-in models
 loopback.autoAttach();
 
+// Get loopback model reference 
+var Model = loopback.Model;
+
+// Create custom models extending Model 
+var CustomModel_1 = Model.extend('CustomModel_1', {}, {
+  trackChanges: true,
```

Comment:
Why is `trackChanges` needed? AFAICT, your test is not using this feature at all.